pages/tutorials/getting_started.md

```diff
@@ -1,61 +1,118 @@
+- To enable the YAML steps editor in Buildkite:
 
-## Add the sample pipeline
+  * Open the [YAML migration settings](https://buildkite.com/organizations/~/pipeline-migration) by selecting _Settings_ > _YAML Migration_.
+  * Select _Use YAML Steps for New Pipelines_, then confirm the action in the modal.
 
-[Pipelines](/docs/pipelines/) are how Buildkite represents your CI workflow, a series of steps that you run on your code.
+- [Git](https://git-scm.com/downloads). This tutorial uses GitHub, but Buildkite can work with any version control system.
 
-Choose a sample pipeline to use as your first pipeline:
+## Understand the architecture
 
-1. Click the appropriate *Add to Buildkite* button:
+Before creating a pipeline, take a moment to understand Buildkite's architecture and the advantages it provides.
 
-    [bash-example test repository](https://github.com/buildkite/bash-example)
+Buildkite uses a hybrid model consisting of the following:
 
-    <a class="inline-block" href="https://buildkite.com/new?template=https://github.com/buildkite/bash-example" target="_blank" rel="nofollow"><img src="https://buildkite.com/button.svg" alt="Add Bash Example to Buildkite" class="no-decoration" width="160" height="30"></a>
+- **Buildkite dashboard:** A software-as-a-service (SaaS) control panel for visualizing and managing CI/CD pipelines. This coordinates work and displays results.
+- **Agents:** Small, reliable, and cross-platform build runners. These are hosted by you, either on-premise or in the cloud. They execute the work they receive from the Buildkite dashboard.
+
+The following diagram shows the split in Buildkite between the SaaS platform and the agents running on your infrastructure.
+
+<%= image "hybrid-saas-architecture.svg", alt: "Shows the hybrid architecture combining a SaaS platform with your infrastructure" %>
+
+The diagram shows that Buildkite provides a web interface, handles integrations with third-party tools, and offers APIs and webhooks. By design, sensitive data, such as source code and secrets, remain within your environment and are not seen by Buildkite. This decoupling provides flexibility and security as you maintain control over the build environment and agent scaling while Buildkite manages the coordination, scheduling, and web interface.
+
+## Install and run an agent
+
+The program that executes work is called an _agent_ in Buildkite. An agent is a small, reliable, and cross-platform build runner that connects your infrastructure to Buildkite. It polls Buildkite for work, runs jobs, and reports results. You can install agents on local machines, cloud servers, or other remote machines. You need at least one agent to run builds.
+
+>📘 Already running an agent
+> If you're already running an agent, skip this section.
```

Comment:
Can we link this directly to the next relevant section?

Reply:
Good idea!

```suggestion
> If you're already running an agent, skip to the [next step](#create-a-pipeline).
```